### ISSUE TITLE
[Feature Request] Ability to move tabs/request body/response body between multiple windows

### ISSUE THREAD

**Issue by Vijayabaskar03** [a85/POSTMan-Chrome-Extension#2813]
It would be great if there is an option to work with multiple windows of Postman, as in moving tabs from one window to another. Even better if the request can be pinned to one window and the response to another. More like a floating tab of the browsers. 

Works great for people like me who use 2 monitors at work.  :)

Thanks

**Comment by vegetableman** [comment]
@Vijayabaskar03 The floating tab is a great idea ^_^. As for being able to see request and response, did you try the two-pane layout?

**Comment by Vijayabaskar03** [comment]
@vegetableman Yes, that's a cool feature and serves almost 90% of the cases, except when the requests and responses are so large, that wrapping them would make it look confusing.

**Comment by javieru** [comment]
It is totally necessary to have multiple windows on Postman. Sometimes you're working on two projects at the same time and you need a set of tabs open in one window and another set in a different window. Please develop feature.
